docs/source/api/apollo-server.mdx

```diff
@@ -457,203 +442,120 @@ The default value is `true`. Set this to `false` to use mocked resolvers only fo
+>
+> * If you're using `startStandaloneServer` you do **not** need to start your server before [passing it directly to `startStandaloneServer`](./standalone).
 
-> This method is provided only by the `apollo-server` package. If you're integrating with Node.js middleware via a different package (such as `apollo-server-express`), instead see both [`start`](#start) and the [framework-specific middleware function](#framework-specific-middleware-function).
+Always call `await server.start()` *before* passing your server into your integration function and starting to accept requests:
 
-Instructs Apollo Server to begin listening for incoming requests:
+<MultiCodeBlock>
 
-```js
-await server.listen({
-  port: 4001,
+```ts
+const server = new ApolloServer<MyContext>({
+  typeDefs,
+  resolvers,
 });
+await server.start();
+
+app.use('/graphql',
+  cors<cors.CorsRequest>(),
+  json(),
+  expressMiddleware(server),
+);
 ```
 
-Takes an `options` object as a parameter, which is passed to the `listen` method of `http.Server`. Supported options are listed in the [documentation for `net.Server.listen`](https://nodejs.org/api/net.html#net_server_listen_options_callback).
+</MultiCodeBlock>
 
-Returns a `Promise` that resolves to an object containing the following properties:
+This allows you to react to Apollo Server startup failures by crashing your process instead of starting to serve traffic.
 
-<table class="field-table">
-  <thead>
-    <tr>
-      <th>Name /<br/>Type</th>
-      <th>Description</th>
-    </tr>
-  </thead>
+If you are testing your server using [`executeOperation`](../testing/testing/) (i.e., you're not actually starting an HTTP server), you don't need to call `start()` because `executeOperation` will do that for you.
 
-<tbody>
-<tr>
-<td>
+### Triggered actions
 
-##### `url`
+The `start` method triggers the following actions:
 
-`string`
-</td>
-<td>
+1. If your server is a [federated gateway](https://www.apollographql.com/docs/federation/managed-federation/overview/), it attempts to fetch its schema. If the fetch fails, `start` throws an error.
+2. Your server calls all of the [`serverWillStart` handlers](/apollo-server/integrations/plugins-event-reference/#serverwillstart) of your installed plugins. If any of these handlers throw an error, `start` throws an error.
 
-The URL that the server is listening on.
-</td>
-</tr>
 
+## `startInBackgroundHandlingStartupErrorsByLoggingAndFailingAllRequests`
 
-<tr>
-<td>
+**Serverless** integrations handle starting an Apollo Server instance in the background by calling the `startInBackgroundHandlingStartupErrorsByLoggingAndFailingAllRequests` method. See [Building integrations](../integrations/building-integrations#ensure-successful-startup) for more details.
 
-##### `server`
+This method [triggers the same actions](#triggered-actions) as the `start()` method.
 
-[`http.Server`](https://nodejs.org/api/http.html#http_class_http_server)
-</td>
-<td>
+## `executeHTTPGraphQLRequest`
```

Comment:
This should probably move lower, next to `executeOperation` and after all the lifecycle methods. Probably should note that this is the main entry point for web framework integrations.